File: 02-read/code.py
## DO NOT RUN
import os

# os.chdir("read/")
##


import pandas as pd

pd.read_csv("data/data1.csv")

pd.read_csv("data-meta/data2.csv")
pd.read_csv("data-meta/data2.csv", skiprows=5)
pd.read_csv("data-meta/data2.csv", skiprows=6)
pd.read_csv("data-meta/data2.csv", skiprows=7)

pd.read_csv("data-meta-delim/data3.csv", skiprows=5)
pd.read_csv("data-meta-delim/data3.csv", skiprows=5, delimiter="$")
pd.read_csv("data-meta-delim/data3.csv", skiprows=5, sep="$")

pd.read_csv("data/data1.csv", skiprows=0, sep=",")
pd.read_csv("data3-same.csv", skiprows=5, sep="$")

File: 02-read/code.R
library(tidyverse)

read_csv('data/data1.csv')

# read_csv('02-read/data/data1.csv')

# read_csv("/Users/dan/git/hub/dsci-100-code_examples/02-read/data/data1.csv")

read_csv('data-meta/data2.csv')

read_csv('data-meta/data2.csv', skip = 6)

read_delim('data-meta-delim/data3.csv', skip = 6)

read_delim('data-meta-delim/data3.csv', skip = 6, delim = '$')

read_delim('data3-same.csv', skip = 6, delim = '$')

